### PR TITLE
EZ-393 Integrating header in pattern lab

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

**Comment by kuldev** [comment]
Looks good, except some bootstrap classes.

File: templates/block/block--system-branding-block.html.twig [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <h2{{ title_attributes.addClass('block-title') }}>{{ label }}</h2>
+  {% endif %}
+  {{ title_suffix }}
+
+  {% block content %}
+    {% include '@atoms/04-images/00-logo.twig' with data %}
+  {% endblock %}
+</section>
+
+
```
Comment (vaibhavjain-in) [inline templates/block/block--system-branding-block.html.twig:77]: Lets remove this Extra line.

File: templates/includes/header.html.twig [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<header id="header" class="header" role="banner" aria-label="{{ 'Site header'|t}}">
+  {% block head %}
+    {% if page.secondary_menu or page.top_header or page.top_header_form %}
+      <nav{{ navbar_top_attributes.addClass('ezcontent-header') }}>
+      {% if container_navbar %}
+      <div class="container tt">
```
Comment (vaibhavjain-in) [inline templates/includes/header.html.twig:6]: I am not sure on the class name `tt`

File: templates/includes/header.html.twig [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+          {% endif %}
+      {% if container_navbar %}
+      </div>
+      {% endif %}
+      </nav>
+    {% endif %}
+    <nav{{ navbar_attributes.addClass('ezcontent-header') }}>
+      {% if container_navbar %}
+      <div class="container ttt">
```
Comment (vaibhavjain-in) [inline templates/includes/header.html.twig:22]: Same here, class name `ttt`. Is this correct ?

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@surbhi90 Some minor feedback, overall we are good to merge.

File: templates/block/block--system-branding-block.html.twig (hunk 1 above)
Comment (surbhi90) [reply templates/block/block--system-branding-block.html.twig]: Fixed

File: templates/includes/header.html.twig (hunk 2 above)
Comment (surbhi90) [reply templates/includes/header.html.twig]: Removed this class

File: templates/includes/header.html.twig (hunk 3 above)
Comment (surbhi90) [reply templates/includes/header.html.twig]: Removed this class also

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**